sections/semantics-embedded-content.include

```diff
@@ -3366,33 +3366,35 @@
 
     <img src="images/home.png" alt="A house icon next to the word 'home'." width="100" height="38" />
 
-    <pre highlight="html">
-      &lt;a href="home.html"&gt;&lt;img src="home.gif" width="15" height="15" alt=""&gt;Home&lt;/a&gt;
-    </pre>
+    <xmp highlight="html">
+      <a href="home.html"><img src="home.gif" width="15" height="15" alt="">Home</a>
+    </xmp>
 
     Where images are used in this way, it would also be appropriate to add the image using CSS.
 
-    <pre highlight="css">
+    <xmp highlight="css">
       #home:before
       {
-      content: url(home.png);
+        content: url(home.png);
       }
-
-      &lt;a href="home.html" id="home"&gt;Home&lt;/a&gt;
-    </pre>
+    </xmp>
+    
+    <xmp highlight="html">
+      <a href="home.html" id="home">Home</a>
+    </xmp>
   </div>
 
     <div class="example">
   In this example, there is a warning message, with a warning icon. The word "Warning!" is in emphasized
   text next to the icon. As the information conveyed by the icon is redundant the <{img}> element is given an empty <code>alt</code> attribute.
 
-  <img src="images/warning.png" alt="Warning!" width="38" height="34" /> <strong>Warning!</strong> Your session is about to expire.
+  <img src="images/warning.png" alt="Warning!" width="38" height="34"> <strong>Warning!</strong> Your session is about to expire.
```

Comment:
Are we removing or adding the closing tag to img? (several cases on this page where the closing tag was removed)